radicle/src/cob.rs

```diff
@@ -21,6 +33,20 @@ pub struct Author {
     did: Did,
 }
 
+impl From<Did> for Author {
+    fn from(did: Did) -> Self {
+        Self { did }
+    }
+}
+
+impl From<NodeId> for Author {
```

Comment:
What would be a circumstance when a Node be a Author?

Reply:
When a user publishes content from his own node, then his node id will be the author of that content.